docs/sources/docker-hub-enterprise/adminguide.md

```diff
@@ -1,104 +0,0 @@
-no_version_dropdown: true
```

Comment:
Thanks, I think it would be good to have redirects added back in at some point. There will be many links pointing to the docs (both from the issue tracker, and external sites).

So "+1" on skipping that in this PR, but it should definitely be on the roadmap.

Would it be possible to add a placeholder/example file containing a redirect for illustration (how it should be done)?